packages/react-components/src/components/ChatMessage/ChatMessageComponentAsEditBox.tsx

```diff
@@ -41,15 +41,18 @@ export type ChatMessageComponentAsEditBoxProps = {
   inlineEditButtons: boolean;
 };
 
+type MessageState = 'OK' | 'too short' | 'too long';
```

Comment:
It messes with my brain to see a space in a string literal type. There's nothing wrong with it, and this is internal only.

But... it messes with me :P

Reply:
hehe